### PR TITLE
area_dgb

### PR DESCRIPTION
**Pull request by Kelebek1**
Beep beep, big commit coming through. Any .git size queens????

### CODE REVIEW

File: ver/us/splat.yaml [hunk 1]
```diff
@@ -8107,14 +8107,14 @@ segments:
...
-    - [0xC59810, rodata]
-    - [0xC59820, rodata]
+    - [0xC55F40,      .c, C55F40]
+    - [0xC56EF0,      .c, C56EF0]
+    - [0xC571B0,   .data, C55F40]
+    - [0xC572A0,   .data, C55F40]
+    - [0xC574C0,   .data, C55F40]
+    - [0xC59620,   .data, C55F40]
+    - [0xC59810, .rodata, C55F40]
+    - [0xC59820, .rodata, C55F40]
```
Comment (nanaian) [inline ver/us/splat.yaml:8122]: Why indent these like this?

Comment (Kelebek1) [reply ver/us/splat.yaml:8122]: Just so they line up. No real reason.

File: src/code_EB8E90.c [hunk 2]
```diff
@@ -200,4 +200,4 @@ void N(func_80240B14_EB9964)(ScriptInstance* script, NpcAISettings* aiSettings,
 
 INCLUDE_ASM(s32, "code_EB8E90", func_80240BD4_EB9A24);
 
-INCLUDE_ASM(s32, "code_EB8E90", func_80240EFC_EB9D4C);
+INCLUDE_ASM(s32, "code_EB8E90", func_80240EFC_EB9D4C);
```
Comment (ethteck) [inline src/code_EB8E90.c:203]: newlines at end of files please

**Review by ethteck — APPROVED** [review]
Other than the newline thing, lgtm!

**Review by nanaian — CHANGES_REQUESTED** [review]
Can you move `new_lines.py` into `tools`?

**Comment by Kelebek1** [comment]
> 
> 
> Can you move `new_lines.py` into `tools`?

I put it with format.sh since it's meant to do the same sort of thing, just format files, but I'll move it.

**Pull request merged**